tests/Task.test.ts

```diff
@@ -129,6 +129,34 @@ describe('parsing', () => {
         ).toStrictEqual(true);
         expect(task!.blockLink).toEqual(' ^my-precious');
     });
+
+    it('supports tag anywhere in the description and separates them correctly from signifier emojis', () => {
+        // Arrange
+        const line =
+            '- [ ] this is a task due 📅 2021-09-12 #inside_tag ⏫ #some/tags_with_underscore';
+        const path = 'this/is a path/to a/file.md';
+        const sectionStart = 1337;
+        const sectionIndex = 1209;
+        const precedingHeader = 'Eloquent Section';
```

Comment:
These values (`path` etc) are not needed for the test... It would be clearer if the tests could use this `fromLine()`:

```typescript
import { fromLine } from './TestHelpers';
```